nixos/modules/services/networking/syncthing.nix

```diff
@@ -16,6 +16,14 @@ in {
         available on http://127.0.0.1:8384/.
       '';
 
+      guiAddress = mkOption {
+        type = types.string;
+        default = "127.0.0.1:8384";
+        description = ''
+          Override GUI listen address.
```

Comment:
How about "Address to server the GUI on" instead? The fact that it overrides the default shouldn't be in the options description

Reply:
Done